src/other/BottomNav.vue

```diff
@@ -35,29 +35,28 @@
 </template>
 
 <script>
+const { mapState } = require('vuex');
 
 export default {
   props: ['dark'],
   data() {
     return {
       bottomNav: 'Admin Page',
-      coach: true,
-      manager: true,
-      admin: true,
     };
   },
   computed: {
+    ...mapState('auth', { user: 'user' }),
```

Comment:
U can access store directly using this.$store.state.auth.user, maybe that would save u writing 1 line instead of 2

Reply:
Its safer to do this as has less of a reference chain for js to call. Your way requires 5 memory calls, mine requires one